coder_rust/square_root.rb

```diff
@@ -1,14 +1,21 @@
 # require 'rspec/autorun'
 
-def square_root(n)
-  return 4
+require 'prime'
```

Comment:
Is this library being used anywhere? 🤔

Reply:
oops you are right! Left it while trying to implement another approach.